### PR TITLE
Added CLI command for generating a QC report for CDM

### PR DESCRIPTION
**Pull request by mhkc**
Added a CLI command for generating the a QC report for CDM in JSON format. The PR includes a unit tests for the command.

### Run 

Run with `prp create-cdm-input -q quast.tsv -p bwa.qc -c chewbbaca.out sample_name`

### Test

`pytest tests`

### CODE REVIEW

**Comment by ryanjameskennedy** [comment]
I was thinking that we could run prp just once like so:
```
prp create-output \\
      --sample-id ${sampleName} \\
      ${amrfinderArgs} \\
      ${brackenArgs} \\
      ${cgmlstArgs} \\
      ${mlstArgs} \\
      ${mykrobeArgs} \\
      ${postalignqcArgs} \\
      ${quastArgs} \\
      ${resfinderArgs} \\
      ${runInfoArgs} \\
      ${tbprofilerArgs} \\
      ${virulenceArgs} \\
      --qc-results ${qcOutput} \\
      --analysis-results ${output}
```
This way we don't need to parse the outputs twice?

**Comment by mhkc** [comment]
I think its cleaner to keep them as separate commands as it makes it clearer what type of input is used to generate which output. CDM is also a proprietary software that other sites are not going to use. Its cleaner to not generate CDM input data by default.

**Comment by ryanjameskennedy** [comment]
Yeah but the `export_to_cdm` is just for creating the arguments file for the upload:
```
echo ${sequencingRun} \\
         -sample-id ${sampleName} \\
         -assay ${species} \\
         -qc ${qc} \\
         ${limsID}
```
Anyways I will create another process in the prp module if you think this is better.

**Comment by ryanjameskennedy** [comment]
Perhaps we could add an `--output` arg?

File: prp/cli.py [hunk 1]
```diff
@@ -86,7 +86,7 @@ def cli():
 @click.option("-t", "--tbprofiler", type=click.File(), help="tbprofiler results")
 @click.option("--correct_alleles", is_flag=True, help="Correct alleles")
 @click.argument("output", type=click.File("w"))
-def create_output(
+def create_bonsai_input(
```
Comment (ryanjameskennedy) [inline prp/cli.py:89]: Does this mean the subarg changes from `create-output` to `create-bonsai-input`? I.e. `prp create-bonsai-input $args`?

**Review by ryanjameskennedy — APPROVED** [review]
LGTM!

**Pull request merged**